packages/p2p/src/components/orders/order-table/order-table-buy-row.jsx

```diff
@@ -0,0 +1,51 @@
+import { Table }                  from 'deriv-components';
+import { localize }               from 'deriv-translations';
+import PropTypes                  from 'prop-types';
+import React                      from 'react';
+import { getFormattedDateString } from 'Utils/date-time';
+
+// TODO: [p2p-refactor-row-component] - Reuse row component that we have in other places
+const BuyOrderRowComponent = React.memo(({ data, onOpenDetails, style }) => {
+    const {
+        counterparty,
+        display_transaction_amount,
+        display_offer_amount,
+        display_status,
+        order_id,
+        order_purchase_datetime,
+        offer_currency,
+        transaction_currency,
+    } = data;
+
+    return (
+        <div style={ style }>
+            <Table.Row>
+                <Table.Cell><span>{ localize('Buy') }<br /><a onClick={ () => onOpenDetails(data) } className='link'>{ order_id }</a></span></Table.Cell>
+                <Table.Cell>{ display_status }</Table.Cell>
+                <Table.Cell>{ transaction_currency }{ ' ' }{ display_transaction_amount }</Table.Cell>
```

Comment:
```suggestion
                <Table.Cell>{`${transaction_currency} ${display_transaction_amount}`}</Table.Cell>
```

Reply:
@mustofa-binary the whitespace might be missed when reading it.

Reply:
After discussion, it was decided that we'll keep this for template code.